package.json

```diff
@@ -76,15 +76,15 @@
   "devDependencies": {
     "@danielx/civet": "0.7.4",
     "@danielx/hera": "^0.8.13",
-    "@prettier/sync": "^0.3.0",
```

Comment:
Do we still need prettier/sync?

Reply:
Yes, we still use it when doing Markdown processing of `<Playground>` at build time, because that interface needs to be synchronous.